sage.py

```diff
@@ -104,105 +97,78 @@ def load_IANA_mapping():
                     "name": row[0] if row[0] else "unknown",
                     "description": row[3] if row[3] else "---",
                 }
-        else:
-            # Do nothing 
-            pass
     return ports
-    
+
+
 ## 3
 def readfile(fname):
-    unparsed_data = None
     with open(fname, 'r') as f:
         unparsed_data = json.load(f)
         
     unparsed_data = unparsed_data[::-1]
-    #print('# events: ', len(unparsed_data))
-    #print(unparsed_data[0])
     return unparsed_data
     
 
-#cats = dict()
-#ips = dict()
-#hosts = dict()
-#h_trig = []
-
 def parse(unparsed_data, alert_labels=[], slim=False):
-    
+
     FILTER = False
```

Comment:
I have kept all such capitalised parameters for now. They can be extracted as global parameters in another PR (if needed)